server/plugin/webhook.go

```diff
@@ -589,7 +589,7 @@ func (p *Plugin) postIssueEvent(event *github.IssuesEvent) {
 			continue
 		}
 
-		if sub.IssueCreations() && action != actionOpened {
+		if sub.IssueCreations() && action == actionClosed {
```

Comment:
2/5 I like the previous code as it ensures that even when new actions are added, the code is only expected for the action open.